specification/e-referrals-service-api.yaml

```diff
@@ -116,7 +116,6 @@ info:
     - when authenticating a human user is not possible
     - for endpoints that offer read-only functionality
     - by ‘Provider Organisations’ (currently)
-    - for accessing endpoints [A004](#api-Default-a004-retrieve-reference-data), [A005](#api-Default-a005-retrieve-referral-request), [A006](#api-Default-a006-retrieve-attachment), [A007](#api-Default-a007-retrieve-clinical-information), [A024](#api-Default-a024-retrieve-advice-and-guidance), [A025](#api-Default-a025-retrieve-advice-and-guidance-conversation) and [A043](#api-Default-a043-retrieve-advice-and-guidance-overview-pdf) (currently)
```

Comment:
Was this deleted fully intentionally or was it meant to be moved somewhere else?

Reply:
Good spot - shouldn't have been deleted